src/generators/notebookgenerator/itemrenderer.tsx

```diff
@@ -23,8 +178,16 @@ export function notebookItemRenderer(
       let collapsed = cellRef!.model.metadata.get(
         'toc-hr-collapsed'
       ) as boolean;
-      collapsed = collapsed != undefined ? collapsed : false;
+      collapsed = collapsed !== undefined ? collapsed : false;
```

Comment:
Ooh, I'd really like to turn on `strictNullChecks` for this project.

Reply:
Yeah, agreed. However, this sort of thing (i.e., loose equality tests) is present throughout the codebase. Such a change should be part of a separate PR aimed at improving code quality and will likely require a bit of work. I had initially tried to clean up a couple other files, but gave up once I started encountering compiler errors and came to see it to as a time sink.